README.md

```diff
@@ -1,2 +1,141 @@
 # mx-chain-ws-connector-template-go
-Template for WebSocket connector written in go
+
+## Introduction
+
+In today's rapidly evolving blockchain ecosystem, the demand for real-time access to exported data from blockchain nodes
+has significantly increased. To meet this growing need within our expanding system, we have developed the Outport
+Driver, a powerful websocket connection system specifically designed for the Multiversx blockchain.
+
+The Outport Driver acts as a crucial bridge between the Multiversx blockchain node and various services that rely on
+receiving up-to-date data from the node. By establishing a websocket connection, the Outport Driver enables seamless
+transmission of essential information, including blocks, validator details, and much more.
+
+One of the key features of the Multiversx blockchain node is its flexible configuration file. By simply setting a flag
+to "true" within this file, the node can initiate the delivery of data via a websocket connection to any connected
+websocket client. This capability provides a reliable and efficient method for receiving live updates from the
+Multiversx blockchain.
```

Comment:
Multiversx -> MultiversX (in the entire file)

Reply:
Right, changed it